Makefile.include

```diff
@@ -730,7 +730,7 @@ debug-server:
 	$(call check_cmd,$(DEBUGSERVER),Debug server program)
 	$(DEBUGSERVER) $(DEBUGSERVER_FLAGS)
 
-emulate:
+emulate: all
```

Comment:
Do we really want to rebuild the firmware each time we call emulate ? I see the emulate target as a mechanism to boot a virtual abstraction of the hardware, similar to the term target on native but maybe it makes more sense to behave like `flash` target (which also depends on `all`). In this case, we could also have an `emulate-only` target to skip the build ?

Reply:
Otherwise we need to change the script a lot in order to flash the emulation. That would be a good improvements, since restarting the emulation takes some time. But it takes some time to implement this improvement. At least, that is my perspective. I would keep it that way, for now. It is a good idea for an improvement though. What do you say @basilfx

Reply:
I would say that this change makes sense. Just like `make flash`, you want to get started. Other tools like `make debug` or `make term` attach to a running target (I could even see `make debug` attach to Renode in the future).

I think this makes sense from a usability point of view :)